src/fields/Resources/ResourcesItem/ResourcesItem.jsx

```diff
@@ -7,29 +7,38 @@ import dateFormat from '../../../definitions/date-format';
 import './resources-item.scss';
 
 function getFileType(mime, fieldSchema) {
-  const type = fieldSchema.enum.indexOf(mime) > -1 ? fieldSchema.enumNames[fieldSchema.enum.indexOf(mime)].toLowerCase() : 'overig';
+  const type = fieldSchema.enum.indexOf(mime) > -1 ? fieldSchema.enumNames[fieldSchema.enum.indexOf(mime)] : 'Anders';
   return (
     <span className={`resources-item__file-type
-      resources-item__file-type--${type}`}
+      resources-item__file-type--${type.toLowerCase()}`}
     >{type}
     </span>
   );
 }
 
-const ResourcesItem = ({ resource, schemaProps }) => (
-  <div className="resources-item">
+const ResourcesItem = ({ resource, schemaProps }) => {
+  let fileType;
+  if (resource['ams:distributionType'] === 'api') {
+    fileType = getFileType(resource['ams:serviceType'], schemaProps['ams:serviceType']);
+  } else if (resource['ams:distributionType'] === 'file') {
+    fileType = getFileType(resource['dct:format'], schemaProps['dct:format']);
+  } else {
+    fileType = getFileType(resource['ams:distributionType'], schemaProps['ams:distributionType']);
+  }
+
+  return (<div className="resources-item">
```

Comment:
Eslint hoort hierover te vallen. Alleen is er geen test stap in de pipeline die dat nu test. Kan je lokaal even kijken of alles goed is. (en wel goed idee om een keer de CI straat goed in te richten voor dit project).